sdk/bento/graph/ecs.py

```diff
@@ -248,8 +239,9 @@ class GraphEntity(Entity):
     """
 
     def __init__(self, components: Iterable[str]):
-        # TODO(zzy): obtain id from actual entity
-        self.id = randint(1, int(1e5))
+        # TODO(zzy): obtain id from actual entity in engine
+        # compute entity id from hash of components to make entity id deterministic
+        self.id = crc32(",".join(sorted(components)).encode())
```

Comment:
I don't think the entity ID should be deterministic. If I have two entities with the same components and same values in their components, it doesn't mean that there is only one entity.

Reply:
This will be eventually replaced with Entity Id from set by the Engine. Right now I need it be deterministic in order to write tests.